### ISSUE TITLE
Calling "step-over" on the last line of the function failed

### ISSUE THREAD

**Issue by WNsACE** [bytecodealliance/wasm-micro-runtime#3475]
Here is the assembly of test code
![image](https://github.com/bytecodealliance/wasm-micro-runtime/assets/26028060/cb04ef26-6f77-4710-bc8c-92c23e62cae5)

When the breakpoint is triggered at "fun [79] return", as shown in the following figure:
![image](https://github.com/bytecodealliance/wasm-micro-runtime/assets/26028060/72bdb19f-17de-41b6-a916-26f1cc6f94d1)

At this point, the runtime is in the following code:
![image](https://github.com/bytecodealliance/wasm-micro-runtime/assets/26028060/738486be-4298-4810-b7da-5f4165aeb518)

When the user calls "step-over", the following code will be triggered
![image](https://github.com/bytecodealliance/wasm-micro-runtime/assets/26028060/c981d475-32a0-4ff5-83e4-228163289dc1)

But the value of "step_count" is 1, so the pause signal will not be triggered
![image](https://github.com/bytecodealliance/wasm-micro-runtime/assets/26028060/a162ace2-4705-4b7b-aa9d-621a257f495d)

At the same time, the value of "step_count" is 1, so the pause signal will not be triggered. However, "return_func" restores the stack, so only the next section of code can be executed. If the next line of code is a calling function, it will enter the function and cause debugging information errors.

**May I ask why "step_count" needs to be equal to 2? And how to resolve the appeal issue?**

**Comment by TianlongLiang** [comment]
I will take a look at this issue, and keep you updated

**Comment by TianlongLiang** [comment]
You are right, this line should be a typo, it should be 1, I submitted PR #3583 for this issue. Thanks for spotting it and reporting it!
